lemur/plugins/lemur_entrust/plugin.py

```diff
@@ -114,9 +114,16 @@ def get_client_id(session, organization):
         # catch an empty json object here
         d = {'response': 'No detailed message'}
 
+    if 'status' in d and d['status'] >= 300:
+        error_messages = d['errors']
+        raise Exception(f"Error for Getting Organization {error_messages}")
+
+    if 'organizations' not in d:
+        raise Exception("Error for Getting Organization: not org returned")
+
     found = False
     for y in d["organizations"]:
-        if y["name"] == organization:
+        if y["name"] == organization and y["verificationStatus"] == 'APPROVED':
```

Comment:
There is a typo few lines after 
`raise Exception(f"Error on Organization - use one from the list: {d['organizations']}")`